src/intervals/macro_definitions.jl

```diff
@@ -32,7 +32,3 @@ end
 macro biginterval(expr1, expr2...)
     make_interval(BigFloat, expr1, expr2)
 end
-
-macro make_interval(T, expr1, expr2...)
```

Comment:
Why did you remove this macro? It allows to specify the type of the interval desired.